### PR TITLE
Finish functionality

### PR DESCRIPTION
**Pull request by m4v15**
relates to #128 #129

### CODE REVIEW

File: text.js [hunk 1]
```diff
@@ -61,12 +63,14 @@ module.exports = {
     // Services page
     serviceQuesion: 'ما نوع الخدمة او الخدمات التي تبحث عنها ',
     food: 'طعام',
-    shopping: 'مشتريات',
-    religion: 'ديانات',
+    sport: 'Sport',
```
Comment (Suuuuha) [inline text.js]: change it to arabic

File: routes/serviceInfo.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
-  latlng('suha lifted her arm', (err, result) => {
-    if (err) res.redirect('/')
+  const id = req.query.id
+  dbFunctions.findById(dbFunctions.Business, id, (error, service) => {
+    if (error) res.render('notFound')
     else {
-      res.render('serviceInfo', {
-        service: mockResult[0],
-        latlng: result
+      // coordinates means إحداثيات
```
Comment (Suuuuha) [inline routes/serviceInfo.js]: do we need to keep this comment in here?

File: text.js [hunk 3]
```diff
@@ -61,12 +63,14 @@ module.exports = {
...
+    education: 'Education',
```
Comment (Suuuuha) [inline text.js]: this as well :)

File: text.js [hunk 4]
```diff
@@ -22,12 +22,14 @@ module.exports = {
...
     health: 'Health',
-    viewPoints: 'View Points',
-    nature: 'Nature',
-    museums: 'Museums',
+    municipal: 'Municipal',
+    fashion: 'Fashion',
+    construction: 'Construction',
+    'it': 'IT',
```
Comment (Suuuuha) [inline text.js]: what did you put it in ' ' ?

File: routes/serviceInfo.js (hunk 2 above)
Comment (MohamedOmarii) [inline routes/serviceInfo.js]: to know what is means :+1:

**Comment by esraajb** [comment]
love the address

**Pull request merged**